tests/unit/php/Core/Framework/Api/EventListener/ErrorResponseFactoryTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Shopware\Tests\Unit\Core\Framework\Api\EventListener;
+
+use Shopware\Core\Framework\Api\EventListener\ErrorResponseFactory;
+
+/**
+ * @internal
+ *
+ * @covers \Shopware\Core\Framework\Api\EventListener\ErrorResponseFactory
+ */
+class ErrorResponseFactoryTest extends \PHPUnit\Framework\TestCase
+{
+    public function testStackTraceForExceptionInDebugMode(): void
+    {
+        $factory = new ErrorResponseFactory();
+
+        /* @var \Symfony\Component\HttpFoundation\JsonResponse $response */
+        $response = $factory->getResponseFromException(new \Exception('test'), true);
+        $data = json_decode($response->getContent());
+        static::assertArrayHasKey('stack', $data);
+        static::assertIsArray($data['stack'], $data);
+
```

Comment:
Please help to execute ECS Fix to solve some basic coding convention issues automatically.

```
composer run-script ecs-fix
```

Reply:
@trandangtri done